src/components/ImageCircle/ImageCircle.sass

```diff
@@ -1,6 +1,6 @@
 @import '../../styles/_partials/index'
 
-.ImageCircleContainer
+body .ImageCircleContainer
```

Comment:
This is an unfortunate workaround having to deal with `styles` added by a developer. There's certainly a better solution, but I didn't want it to slow down progress.

Reply:
Yeah, I'd like to see this eliminated if possible.

Reply:
Good call. Done. Added some props to remove the need for this.